doc/man1/flux-resource.rst

```diff
@@ -109,6 +109,56 @@ snippet for an existing named format may be generated with
 ``--format=get-config=NAME``.  See :man1:`flux-jobs` *OUTPUT FORMAT*
 section for a detailed description of this syntax.
 
+The field names that can be specified for the **status** and **drain**
+commands are:
+
+**state**
+   State of node(s): "online", "offline", "avail", "exclude", "drain",
+   "draining", "drained", "all"
+
+**nnodes**
+   The number of nodes in state.
```

Comment:
I don't think this is just the number of nodes in a given state. It reports the number of nodes in the line being printed. If the format is only `{nnodes}`, then the total number of nodes is printed, e.g.
```
$ flux resource list -no {nnodes}
121
```

I'd suggest it is clearest to just say "The number of nodes"

Reply:
BTW, this is how `flux resource info` works, it is just 
```
$ flux resource list -no "{nnodes} Nodes, {ncores} Cores, {ngpus} GPUs"
121 Nodes, 5808 Cores, 968 GPUs
```

Reply:
ahh, i see that there is a `resources_uniq_lines()` function that alters the output dependent on how many lines there will be.  I wasn't aware of that quirk.

Reply:
Yeah, I wonder if that should be added to the documentation for `flux resource list`, `status` and `drain`. I'm struggling with a concise description of what the commands do though, maybe something like "These commands combine resources into a single line of output when possible depending on the supplied output format. Resource counts are not included in the determination of uniqueness."